webpack.config.js

```diff
@@ -16,6 +16,13 @@ module.exports = {
           loader: 'babel-loader',
         },
       },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
+          'css-loader'
+        ]
+      }
```

Comment:
I  had to configure without which the component was not picking up css even though the .css file in the same package as component

Reply:
Understood, now we can use individual stylesheets for our widgets.